insights/parsers/sshd_test_mode.py

```diff
@@ -0,0 +1,52 @@
+"""
+SshdTestMode - command ``sshd -T``
+==================================
+
+This module contains parsers that check the output of "/usr/sbin/sshd -T" command.
+
+"""
+from insights.core import Parser
+from insights.core.plugins import parser
+from insights.parsers import get_active_lines
+from insights.specs import Specs
+
+
+@parser(Specs.sshd_test_mode)
+class SshdTestMode(Parser, dict):
+    """
+    This parser reads the output of "/usr/sbin/sshd -T" command.
+
+    Sample output::
+
+        port 22
+        addressfamily any
+        listenaddress [::]:22
+        listenaddress 0.0.0.0:22
+        usepam yes
+        logingracetime 120
+        x11displayoffset 10
+        x11maxdisplays 1000
+        maxauthtries 6
+        ciphers aes256-gcm@openssh.com,chacha20-poly1305@openssh.com,aes256-ctr,aes128-gcm@openssh.com,aes128-ctr
+        macs hmac-sha2-256-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha2-256,hmac-sha1,umac-128@openssh.com,hmac-sha2-512
+
+    Examples:
+        >>> len(sshd_test_mode)
+        10
+        >>> sshd_test_mode.get("addressfamily")
+        'any'
+        >>> sshd_test_mode.get("ciphers")
```

Comment:
Hi @wushiqinlou , as we discussed , we can see to add a note in this parser's docstrings, to tell the parser users to take care of the data type, eg. `string` or `list`, when getting data.

Reply:
@wushiqinlou - Per our discussion, I updated the value format as list for all keys, please check

Reply:
Thanks @wushiqinlou , the changes looks good to me. 👍